test/e2e/opentelemetry/app/api/app/[param]/data/edge/route.ts

```diff
@@ -1,4 +1,7 @@
 export async function GET() {
+  // ensure performance is available in edge
+  console.log(performance.now())
```

Comment:
A better test would be to do this as the module's side-effect. Because different packages polyfill `performance` thus masking the problem once "everything is already imported".

Reply:
Updated